Make Text Static on Single Season

JPKribs opened the pull request:
### Summary

Resolves: https://github.com/jellyfin/Swiftfin/issues/1577

Make the `SeasonSelector` static Text for single Season shows.

**Notes:**
- I mirrored the regular `SeasonSelector` spacing. I'm not sure if that's a good fit so it might be worth removing the horizontal spacing.
- I made this text also accent color. It looks good as `.primary` as well so we have alternatives. 

### Screenshot

<details>

<summary>Single vs Multiple Season(s)</summary>

<img width="450" alt="Screenshot 2025-06-16 at 08 36 50" src="https://github.com/user-attachments/assets/3d5cbcf9-bb57-44c7-b4b6-35de1a7096d6" />

</details>

LePips reviewed (changes requested):
I would like the text to be like Apple TV, where the static text would be larger and would be the text primary color. The accent color should largely be for "this can be selected".

Please also remove the padding.

JPKribs commented:
> I would like the text to be like Apple TV, where the static text would be larger and would be the text primary color. The accent color should largely be for "this can be selected".
> 
> Please also remove the padding.

Here is my revised version. I used the same font & weight as the `PillHStack` headers to match that:

![Simulator Screenshot - iPhone 16 Pro - 2025-06-16 at 23 10 43](https://github.com/user-attachments/assets/6d483446-74cf-41a3-a728-c5f002760379)

sjorge commented:
I did a build an was having a look as I have a lot of 1 season shows (anime) but it still showed the drop down.

After looking at the code it seems this only removes the drop downs when the seasons are named. Was this intentional?

JPKribs commented:
No but also having the text just be "-" also isn't ideal. But, I'm not really sure how you can have a season without a name. That value isn't optional and we have a fallback:

https://github.com/jellyfin/Swiftfin/blob/main/Shared/Extensions/JellyfinAPI/BaseItemDto/BaseItemDto.swift

It should always be a name or "Unknown"? Do you have empty season/specials folders? I know Jellyfin still counts those as seasons.

LePips commented:
Frankly, I'm unable to reproduce the issue with my one season anime.

JPKribs commented:
@sjorge please try: https://github.com/jellyfin/Swiftfin/pull/1591.

I moved the season text to a computer value and removed the `if let` for the season name. Instead, I'm just basing this on total list of seasons in the `ViewModel`. So this should work even if the selection starts off as `nil`.

Still WIP for testing and I want to chain this with a fix for the layout under-assigning the vertical size of the frame:

<img width="272" alt="Screenshot 2025-06-18 at 09 46 34" src="https://github.com/user-attachments/assets/ac322b15-4708-46c1-9e0b-8500aa5b0b96" />
